.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Expected Behavior
+
+### Actual Behavior
+
+### Steps to Reproduce
+
+### Minimal yet Complete Reproducer Code (or URL to Code)
+
+### Netty Version
```

Comment:
For Netty version, you can use something very similar to [this](https://github.com/orientechnologies/orientdb/blob/master/.github/ISSUE_TEMPLATE.md#orientdb-version-operating-system-or-hardware). You can also declare that Netty 3.x is EOL and no longer supported and a person needs to upgrade to 4.0.x or 4.1.x. WDYT?